base/reflection.jl

```diff
@@ -1216,9 +1243,10 @@ function MethodList(mt::Core.MethodTable)
 end
 
 """
-    methods(f, [types], [module])
+    methods(f, [types], [module]) -> AbstractVector{Method}
 
-Return the method table for `f`.
+Return a list of [`Method`](@ref)s for `f`. The returned container type is not specified.
```

Comment:
CI fails because this cross-ref is broken. Probably because the `Method` docstring you added is not in the manual. (You'll have to put it in one of the .md files under `doc/src`)